VideoAPI/Testing.Common/Helper/ZAP.cs

```diff
@@ -9,34 +9,26 @@
 
 namespace Testing.Common.Helper
 {
-    public static class ZAP
+    public static class Zap
     {        
-        private static readonly ClientApi _api = new ClientApi(ZapConfiguration.ApiAddress, ZapConfiguration.ApiPort, ZapConfiguration.Apikey);
-        private static IApiResponse _apiResponse;
+        private static readonly ClientApi _api = new ClientApi(ZapConfiguration.ApiAddress, ZapConfiguration.ApiPort, ZapConfiguration.ApiKey);
 
-        private static ZAPConfiguration ZapConfiguration
+        private static ZapConfiguration ZapConfiguration
         {
             get
```

Comment:
if you're only returning a get then `ZapConfiguration => ` would suffice

Reply:
Done/updated